### ISSUE TITLE
case sensitive iis-app-create.json

### ISSUE THREAD

**Issue by bdaniel7** [OctopusDeploy/Library#267]
This script (iis-app-create.json) must be tested better.
I tried to used it as-is but it failed multiple times, because powershell is case sensitive (at least on my servers).

**Comment by alfhenrik** [comment]
@bdaniel7 thanks for letting us know! 

I've tested it on one of my local environments and it worked correctly so could you please let us know any errors that you are seeing when using this step template in your deployment?

Thanks!

**Comment by bdaniel7** [comment]
I'll get back on Monday with more details (+ a diff between the original and my current version), but in short, the application pool was not set when the application was updated.

At this line:
Set-ItemProperty IIS:\Sites\$parentSite\$name -Name ApplicationPool -Value $applicationPoolName

Once I used "applicationPool" instead of "ApplicationPool", it worked.

**Comment by bdaniel7** [comment]
![Uploading iis-app-create-mine-vs-library.png…]()

**Comment by bdaniel7** [comment]
This is a diff between the library and my corrected version.
![iis-app-create-mine-vs-library](https://cloud.githubusercontent.com/assets/215914/12031727/5953a38e-ae18-11e5-962e-630506303e75.png)

**Comment by engstrumpan** [comment]
The problem seems to have something to do with Windows Server 2012 and/or IIS 8.
Have a [look at the following post](https://forums.iis.net/t/1200253.aspx) where the same issue is experience but not connected to this particular script.

I've tested the the script, in particular the part `Set-ItemProperty IIS:\Sites\$parentSite\$name -Name ApplicationPool -Value $applicationPoolName` on my local Win7 IIS 7.5 machine and it works as intended.
Doing the very same thing on our server machine running Windows Server 2012 with IIS8 this however does not work. 
The output from the following operation doesn't provide an adequate message whether it succeded or not. The lack of information or exception to me tells me it should've worked.
`Set-ItemProperty IIS:\Sites\RandomSite -verbose`
`VERBOSE: Performing operation "Set-ItemProperty" on Target
-name ApplicationPool -value SomeAppPool".`

So after I read the post I linked to I tried to change the property name from ApplicationPool to applicationPool and voila - it worked as expected and my application now got the assigned application pool.
